client/src/adapter/dosDashboardRefresh.ts

```diff
@@ -40,7 +40,8 @@ function parseDiscrepancyCounts(data: any): any {
 
     const total = _.sum(_.values(data));
     const opportunistic = data.OPPORTUNISTIC_BENEFITS || 0;
-    const audited = total - opportunistic;
+    const tied = data.TIED_CONTEST || 0;
+    const audited = total - opportunistic - tied;
 
     return { audited, opportunistic };
```

Comment:
Shouldn't `return { audited, opportunistic }` be `return { audited, opportunistic + tied }`?

Reply:
Maybe. I was thinking we didn't care about tied contests _at all_, since they can't be audited by the current system. But I should make it consistent with whatever you're doing in the reports.

Reply:
I think it makes sense to count them _somewhere_, because a discrepancy is still a discrepancy - they are supposed to record all the choices on the ballots they audit.